phpBB/phpbb/template/twig/loader.php

```diff
@@ -23,6 +23,16 @@ class loader extends \Twig_Loader_Filesystem
 	protected $safe_directories = array();
 
 	/**
+	 * Constructor
+	 *
+	 * @param string|array	$paths
+	 */
+	public function __construct($paths = array())
+	{
+		parent::__construct($paths, filesystem_helper::realpath(__DIR__);
```

Comment:
I still do not understand why would you call `realpath` here.

Reply:
It already was...
Anyway, it was necessary before 5.3.0.
There are many more things that can be improved because old php versions are not supported